### DISCUSSION TITLE
First Ride Thoughts & Suggestions

### DISCUSSION THREAD

**Discussion by iilis** [sandermvanvliet/RoadCaptain#71] [General]
Just had my first ride with RoadCaptain yesterday and had some thoughts/suggestions.  
I’ll number them individually so we can discuss further.

**Builder**
1. Create a screen specifically to edit/save settings.  This could include settings for the Builder and Runner apps.  (Suggested settings in the next section.)

**Runner**

2.  Add option to display measurements in metric (km and m) or imperial (miles and ft).  OR would it be possible to grab this setting from the Zwift settings so it always matches?

3.  Add option to show/hide various elements of the Runner, specifically the elevation ascending and descending values in the header but could be any field (not sure if you want to drop it entirely is all fields are hidden?).

4. It seems to me that the RoadCaptain app works by listening for and sending specific commands to the Zwift app window (on a basic level) since the RoadCaptain Runner window is on top and the Zwift app is running in the background.  _Therefore_, could you send other commands or key actions to the Zwift window?  For example the spacebar to use a power up, the number keys to change the camera angle, F10 to take a screenshot.

I didn’t want to make these Issues in the official list unless you agreed with them.  And I’m willing to discuss these if you’d like.

**Comment by sandermvanvliet** [comment]
Thanks for the feedback 👍
I think there are some solid points here, I’ll give it some thought and get back to you.

**Comment by iilis** [comment]
Sorry, forgot to mention that I actually rather liked the overall experience in using the Runner app!  :)

**Comment by iilis** [comment]
One more suggestion:
In the builder, could you do something about the "Beach road to Zwift KOM" segment?  
It is such a small segment that when trying to select it from adjacent segments, it is hidden by the Red Dot marking the end of the ride and I had trouble clicking on this segment.
Perhaps have a near by block of text (with an arrow) to more easily select that segment?

**Comment by sandermvanvliet** [comment]
Agree that it’s pretty annoying to properly select that one. It’s an artefact of how the segments are created. 
I might see if I can just auto select it instead.
